README.md

```diff
@@ -61,7 +61,7 @@ int main()
 
     // Create a task that awaits the doubling of its given value and
     // then returns the result after adding 5.
-    auto double_and_add_5_task = [&](uint64_t input) -> coro::task<uint64_t> {
+    auto double_and_add_5_task = [${EXAMPLE_CORO_TASK_CPP}](uint64_t input) -> coro::task<uint64_t> {
```

Comment:
looks like something weird happened to the README.md file? does it need to be regenerated?

Reply:
That's weird thing I noticed. It keep happening on a commit due to a pre-commit hook. I'll fix it when I get back on my linux box tomorrow.

Reply:
Done